tests/system/requests/test_download.py

```diff
@@ -58,60 +98,42 @@
         ),
     },
     {
-        u"path": os.path.realpath(os.path.join(DATA_DIR, u"file.txt")),
+        u"path": get_path(u"file.txt"),
         u"content_type": PLAIN_TEXT,
-        u"checksum": u"KHRs/+ZSrc/FuuR4qz/PZQ==",
+        u"checksum": u"XHSHAr/SpIeZtZbjgQ4nGw==",
```

Comment:
just verifying that this isn't a breaking change?

Reply:
Yup.  We are no longer validating checksums for non-raw downloads, so the old "expanded" checksum is not valid.